gammapy/maps/axes.py

```diff
@@ -1885,6 +1945,29 @@ def __init__(self, edges_min, edges_max, reference_time, name="time", interp="li
         if np.any(delta < 0 * u.s):
             raise ValueError("Time intervals must not overlap.")
 
+    @property
+    def is_contiguous(self):
```

Comment:
Yes, this needs a bit more thought on what to do with the `node_type`. However in general the situation is present in any case, because the `TimeMapAxis` can be contiguous, even if it's implicit only. When we keep this `.is_contiguous ` property there is probably no need for a second `node_type`?

Reply:
I guess one question is the API similarity between `MapAxis` and the time and label axes.